README.md

```diff
@@ -13,19 +13,29 @@ FAIMMS/faimms_data_rss_channels_process
 
 # Injected Environment Variables
 
-During deployment, the following environment variables will be available for
-cronjobs, they may or may not be used. Using them will result in more
+During the deployment of data services (see [chef recipe](https://github.com/aodn/chef/blob/eb1535192b526ca775fa557630d3221187e766c7/cookbooks/imos_po/recipes/data_services.rb#L131)), various environment variables are made available for
+cronjobs (they may or may not be used). Using them will result in more
 relocatable and robust scripts.
 
 The environment variables are:
 
-|Name              |Default                    |Purpose                        |
-|------------------|---------------------------|-------------------------------|
-|$ARCHIVE_DIR      |/mnt/ebs/archive           |Archive                        |
-|$INCOMING_DIR     |/mnt/ebs/incoming          |Incoming                       |
-|$WIP_DIR          |/mnt/ebs/wip               |Work In Progress tmp dir       |
-|$DATA_SERVICES_DIR|/mnt/ebs/data-services     |Where this git repo is deployed|
-|$LOG_DIR          |/mnt/ebs/log/data-services |Designated log dir             |
+|Name |Default | Purpose|
+|:--|:--|:--|
+| $ARCHIVE_DIR | /mnt/ebs/archive | Archive |
+| $ARCHIVE_IMOS_DIR | /mnt/ebs/archive | Archive |
+| $INCOMING_DIR | /mnt/ebs/incoming |Incoming |
+| $ERROR_DIR | /mnt/ebs/error |Errors |
+| $WIP_DIR | /mnt/ebs/wip |Work In Progress tmp dir |
+| $DATA_SERVICES_DIR | /mnt/ebs/data-services | Where this git repo is deployed |
+| $DATA_SERVICES_TMP_DIR | nil | |
+| $EMAIL_ALIASES | /etc/incoming-aliases | List of configured aliases |
+| $PYTHONPATH | $DATA_SERVICES_DIR/lib/python | Location of data-services python interpreter |
```

Comment:
There is no python interpreter there - AFAIK these scripts usually run on the system python!
```suggestion
| $PYTHONPATH | $DATA_SERVICES_DIR/lib/python | Location of data-services python scripts/modules |
```